splitio/push/splitsse.py

```diff
@@ -200,16 +200,16 @@ async def start(self, token):
         self.status = SplitSSEClient._Status.CONNECTING
         url = self._build_url(token)
         try:
-            sse_events_task = self._client.start(url, extra_headers=self._metadata)
-            first_event = await anext(sse_events_task)
+            self.sse_events_task = self._client.start(url, extra_headers=self._metadata)
```

Comment:
if the variable needs to be scoped to the instance and not the method, lets initialize to `None`, on the `__init__` method (and check it to have a value before using it on `shutdown()`